File: src/main/java/ru/citeck/idea/languageInjections/CiteckFormLanguageInjection.java
package ru.citeck.idea.languageInjections;

import com.intellij.openapi.extensions.ExtensionPointName;
import com.intellij.util.xmlb.annotations.Attribute;
import lombok.Data;

@Data
public class CiteckFormLanguageInjection {

    public static final ExtensionPointName<CiteckFormLanguageInjection> EP_NAME =
        ExtensionPointName.create("ru.citeck.idea.citeckFormLanguageInjection");

    @Attribute("componentType")
    public String componentType;

    @Attribute("path")
    public String path;
}

File: src/main/kotlin/ru/citeck/idea/languageInjections/CiteckFormLanguageInjection.kt
package ru.citeck.idea.languageInjections

import com.intellij.openapi.extensions.ExtensionPointName
import com.intellij.util.xmlb.annotations.Attribute
import lombok.Data

@Data
class CiteckFormLanguageInjection {

    companion object {
        val EP_NAME: ExtensionPointName<CiteckFormLanguageInjection> =
            ExtensionPointName.create("ru.citeck.idea.citeckFormLanguageInjection")
    }

    @Attribute("componentType")
    lateinit var componentType: String

    @Attribute("path")
    lateinit var path: String
}
